internal/gengapic/client_init.go

```diff
@@ -138,10 +142,10 @@ func (g *generator) clientOptions(serv *descriptor.ServiceDescriptorProto, servN
 func (g *generator) clientInit(serv *descriptor.ServiceDescriptorProto, servName string) error {
 	p := g.printf
 
-	var hasLRO bool
+	var hasLRORpc bool
```

Comment:
It's not great, but this should be `hasLRORPC`. Perhaps `hasRPCForLRO`?

https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
sgtm! Done.